### ISSUE TITLE
Feature request: paste image from the system clipboard

### ISSUE THREAD

**Issue by dandv** [kogmbh/WebODF#916]
I'm [comparing WYSIWYG editors](https://github.com/iDoRecall/comparisons/blob/master/JavaScript-WYSIWYG-editors.md) and have been looking for this feature.

**Comment by vandenoever** [comment]
You can paste with ctrl-v.

**Comment by dandv** [comment]
Nope. Just tried at http://www.webodf.org/demo/ci/wodotexteditor-0.5.8/localeditor.html.

**Comment by vandenoever** [comment]
On what browser and operating system?

**Comment by dandv** [comment]
Chrome 44 on Windows.

**Comment by vandenoever** [comment]
I do not have a windows machine on which I can install chrome, but ctrl-v for pasting works with chromium 40 on linux. It is using the [PasteController](https://github.com/kogmbh/WebODF/blob/97ebc92d93b6c23844d0a2685ddf897b31ac84a2/webodf/lib/gui/PasteController.js). This gets the information from the browser paste events via [SessionController](https://github.com/kogmbh/WebODF/blob/97ebc92d93b6c23844d0a2685ddf897b31ac84a2/webodf/lib/gui/SessionController.js#L268).
